server/package.json

```diff
@@ -4,9 +4,6 @@
   "description": "",
   "author": "",
   "license": "MIT",
-  "engines": {
-    "node": "16.*"
```

Comment:
I know there's some weirdness with getting the precommit hooks to work because the BE is on 16 and the FE is on 12 but unfortunately we can't just get rid of this. Does it work if you put this back and set it to `"16.* || 12.*"`?

Reply:
It didn't work for me locally so I just added these lines back